packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -282,6 +321,43 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
               </IconText>
             </IconInfo>
           </Row>
+          {!isMultipleRel && (
```

Comment:
Should this be added to the Page Attachment as well? (Per Pascal't comment: https://github.com/google/web-stories-wp/issues/4440#issuecomment-963277350)

Reply:
Added.